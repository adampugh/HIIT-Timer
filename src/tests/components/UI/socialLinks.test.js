import React from "react";
import { shallow } from "enzyme"; 

import SocialLinks from "../../../components/UI/socialLinks";

test("should render socialLinks correctly", () => {
    const wrapper = shallow(<SocialLinks />);
    expect(wrapper).toMatchSnapshot();
});