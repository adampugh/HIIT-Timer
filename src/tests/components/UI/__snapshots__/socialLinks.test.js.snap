// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render socialLinks correctly 1`] = `
<div
  className="social-links"
>
  <span
    className="twitter"
  >
    <i
      className="fab fa-twitter"
    />
  </span>
  <span>
    <i
      className="fab fa-soundcloud"
    />
  </span>
  <span
    className="youtube"
  >
    <i
      className="fab fa-youtube"
    />
  </span>
</div>
`;
